R/c14_date_list_order_variables.R

```diff
@@ -35,7 +35,8 @@ order_variables.c14_date_list <- function(x) {
       .data[["c14age"]],
       .data[["c14std"]],
       dplyr::matches("^calage$"),
```

Comment:
Can go away, can't it? `get_CalPal ` should also be adjusted to drop the columns CALAGE and CALSTD.